CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # CHANGELOG
 
 ## Release 2.7.2 (in development)
+### Bug fixes
+- Properly embed `pause` image needed for MetalK8s to work offline
```

Comment:
Maybe add the version number here, and why this is needed (i.e., new `containerd` version)

Reply:
Right